orbit/pkg/bitlocker/bitlocker_management_notwindows.go

```diff
@@ -0,0 +1,20 @@
+//go:build !windows
+// +build !windows
+
+package bitlocker
+
+func GetRecoveryKeys(targetVolume string) (map[string]string, error) {
+	return nil, nil
+}
+
+func EncryptVolume(targetVolume string) (string, error) {
+	return "", nil
+}
+
+func DecryptVolume(targetVolume string) error {
+	return nil
+}
+
+func GetEncryptionStatus() ([]VolumeStatus, error) {
+	return []VolumeStatus{}, nil
```

Comment:
I haven't found a place where `GetEncryptionStatus` is called in the PR (might be for future need?), but this should be safe to return `nil, nil` as a nil slice can safely be used with `len` or `range`, etc.

Reply:
Yes, this was for future usage